docker-compose.yml

```diff
@@ -12,11 +12,11 @@ services:
             PGSQL_DB_1_POSTGIS: "true"
         volumes:
             # Use a volume to enable database persistence across restarts
-            - /tmp/nodewatcher-db:/var/lib/postgresql/9.5/main
+            - ~/nodewatcher-db:/var/lib/postgresql/9.5/main
```

Comment:
Because in order to test this out properly I cant have my DB reset after every boot.
I will revert that back,this way it is easier for me to test out since all the code is equal on remote and local branch